### PR TITLE
docs: [issuance] move credential manifest details inside credential_manifest key in offer-credential message

### PR DESCRIPTION
**Pull request by rolsonquadras**
Add new key for credential manifest data similar to [presentation-definition in request-presentation message](http://localhost:3000/#step-3-send-message-requesting-presentation)

### CODE REVIEW

File: spec/spec.md [hunk 1]
```diff
@@ -606,149 +606,151 @@ In the following message structure, the issuer wants a Permanent Resident Card (
...
-                     "credentialSubject":{
-                        "@explicit":true,
+               "credential_manifest":{
+                  "id":"dcc75a16-19f5-4273-84ce-4da69ee2b7fe",
+                  "version":"0.1.0",
+                  "issuer":{
+                     "id":"did:example:123?linked-domains=3",
+                     "name":"Washington State Government",
+                     "styles":{
+
```
Comment (OR13) [inline spec/spec.md:616]: is there a non empty object we can supply here?

File: spec/spec.md [hunk 2]
```diff
@@ -606,149 +606,151 @@ In the following message structure, the issuer wants a Permanent Resident Card (
...
-                        "VerifiableCredential",
-                        "PermanentResidentCard"
-                     ],
-                     "credentialSubject":{
-                        "@explicit":true,
+               "credential_manifest":{
+                  "id":"dcc75a16-19f5-4273-84ce-4da69ee2b7fe",
+                  "version":"0.1.0",
+                  "issuer":{
+                     "id":"did:example:123?linked-domains=3",
```
Comment (OR13) [inline spec/spec.md:613]: this is not a valid issuer id.

Comment (OR13) [inline spec/spec.md:613]:
```suggestion
                     "id":"did:example:123",
```

File: spec/spec.md [hunk 3]
```diff
@@ -606,149 +606,151 @@ In the following message structure, the issuer wants a Permanent Resident Card (
...
-                        "properties":[
-                           {
-                              "path":[
-                                 "$.donor",
-                                 "$.vc.donor"
-                              ],
-                              "fallback":"Unknown",
-                              "label":"Organ Donor"
+                        "styles":{
+                           "thumbnail":{
```
Comment (OR13) [inline spec/spec.md:729]: maybe reuse these styles? above

**Review by OR13 — CHANGES_REQUESTED** [review]
see suggestion for issuer, beware of did urls they are the path to no interop... especially when applied inconsistently.

File: spec/spec.md (hunk 2 above)
Comment (rolsonquadras) [reply spec/spec.md]: Thanks - [updated](https://github.com/decentralized-identity/waci-presentation-exchange/compare/49ef725a91640149a855098944f6f56ca2d66351..f4de63446640af60cddcf1c4c6ac69b97dc5269d)

Originally, I copied from https://identity.foundation/credential-manifest/#credential-manifest-2.

File: spec/spec.md (hunk 1 above)
Comment (rolsonquadras) [reply spec/spec.md]: [updated](https://github.com/decentralized-identity/waci-presentation-exchange/compare/49ef725a91640149a855098944f6f56ca2d66351..f4de63446640af60cddcf1c4c6ac69b97dc5269d)

File: spec/spec.md (hunk 3 above)
Comment (rolsonquadras) [reply spec/spec.md:742]: dinedone

**Review by brentzundel — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Pull request merged**